kubernetes/kubernetes.libsonnet

```diff
@@ -177,6 +177,7 @@ local letsbuildJob(config, withServiceAccountObject={}) = {
       'sidecar.istio.io/inject': 'false'
     })
     + job.mixin.spec.withBackoffLimit(0)
+    + job.mixin.spec.withTtlSecondsAfterFinished(180)
```

Comment:
When a spot instance gets killed, it gets 120s, right? So would it make sense to use that value here?

Reply:
I fail to see the correlation between those :confused:

Reply:
Discussed offline.